pkg/job/v1alpha3/provider.go

```diff
@@ -2,36 +2,39 @@ package v1alpha3
 
 import (
 	"fmt"
+	"github.com/kubeflow/katib/pkg/apis/controller/trials/v1alpha3"
+	"reflect"
 
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 
-	"github.com/kubeflow/katib/pkg/controller.v1alpha3/consts"
-	"github.com/kubeflow/katib/pkg/job/v1alpha3/job"
-	"github.com/kubeflow/katib/pkg/job/v1alpha3/kubeflow"
 	commonv1 "github.com/kubeflow/tf-operator/pkg/apis/common/v1"
 )
 
+var (
+	ProviderRegistry = make(map[string]reflect.Type)
```

Comment:
Can we use Provider instead of reflect.Type?

Reply:
Done.